src/psyclone/dynamo0p3.py

```diff
@@ -5042,7 +5051,7 @@ class DynKernelArguments(Arguments):
     :type call: :py:class:`psyclone.parse.KernelCall`
     :param parent_call: the kernel-call object.
     :type parent_call: :py:class:`psyclone.domain.lfric.LFRicKern`
-    :param bool check: whether to check for consistency between the \
+    :param bool check: whether to check for consistency between the
         kernel metadata and the algorithm layer. Defaults to True.
```

Comment:
Could you align this with the above line please?

Reply:
This isn't mandated by the style guide - you either indent by (a multiple of) 4 spaces *or* line it up with the previous line, whichever you prefer. In this case I've done the former to save having to split to another line.